rfcs/text/0000-roadmaps+planning.md

```diff
@@ -0,0 +1,421 @@
+
+#### Creating the Engineering Plan
+[Creating the Engineering Plan]:#creating-the-engineering-plan
+
+***Next Release Milestone 1 Week 2***. In consultation with their relevant Technical Leads and collaborators, the Product Owner MUST complete the refinement and modeling for *well-understood* Epics, including the addition of child issues for product readiness and the surfacing of dependencies between software components using ZenHub Dependencies.
+
+Milestones MUST be assigned to all issues assigned to the Release not blocked by a Spike.
+
+It is RECOMMENDED that caution be exercised when scheduling issues in a dependency chain in the same Milestone. Historically, this has been the source of slips in DCP schedules.
+
+The Release Shepherd MUST validate accurate modeling of Objectives for the Release in ZenHub and create a Release epic named “Q#YYYY” such as “Q42019” which contains all the related *dcp* Epics and issues for the Release. This will automatically include all child issues in other component repo(s).
+
+This Epic represents the Engineering Plan which MUST be sent to the DCP PM mailing list for review and approval, noting any Objectives that were cut or reduced in scope due to engineering capacity. The Project Leads MUST respond to the original message within two days and MAY negotiate changes to the Engineering Plan. 
+
+***Next Release Milestone 1 Week 3***. Further refinements for Milestone 2 are completely modeled. All Milestone 1 issues that will slip to Milestone 2 MUST be identified.
+
+***Next Release Milestone 2 Week 3***. Further refinements for Milestone 3 are completely modeled. All Milestone 2 issues that will slip to Milestone 3 MUST be identified.
+
+***Next Release Milestone 3 Week 3***.
+All Milestone 3 issues that will slip to the next Release MUST be identified.
+
+#### Tracking Release Progress
+Product Owners for each Objective MUST ensure that their Epics and child issues are kept up-to-date with current status.
+
+Technical Leads MUST drive implementations of prioritized Epics forward and communicate blockers or changes in schedule if they arise.
+
+Over-communication is strongly RECOMMENDED.
+
+***Every two weeks***. The Release Shepherd facilitates a one hour meeting for Product Owners, User Experience, and Technical Leads to review the progress of the Release in ZenHub and address emerging issues.
+
+The Release Shepherd MUST send a status update to the Project Leads on the DCP PM mailing list that captures action items for participants, documents slips in schedule, and/or requests clarification as needed.
+
+As required, the Release Shepherd SHOULD schedule time on the next DCP PM call to address issues pending resolution. The Project Leads MUST never be surprised by the *state of play*.
+
+***At Milestone Completion***. The Project Leads MUST update Oversight on the progress of the Release to ensure appropriate accountability.
```

Comment:
This might go more smoothly if it's known *a priori* which PL will do this for each milestone. "One (and only one)..." language may be appropriate here.

Reply:
Generally, I prefer the _There can be only one_ (also known as the Highlander option) when we need one specific owner - for example, if there are four owners for an epic then who is responsible for getting it to "done"? In the case of the Oversight review, I was trying not to over-specify or over-constrain the Project Leads. I could imagine a scenario where individual Project Leads spoke to their favorite epics like AUDR or Mouse data in turns.

Reply:
I agree more detail here may have diminishing returns, but @ambrosejcarr I think the point of the PLs understanding how we do this as a group is important but I am not sure it will be contentious.

Reply:
@ambrosejcarr - may I resolve this conversation?